src/main/java/net/javadiscord/javabot/systems/help/HelpListener.java

```diff
@@ -54,40 +48,58 @@ public class HelpListener extends ListenerAdapter implements ButtonHandler {
 		newThreadChannels = new HashSet<>();
 	}
 
+	private final UserPreferenceService preferenceService;
 	private final BotConfig botConfig;
 	private final HelpAccountRepository helpAccountRepository;
 	private final HelpTransactionRepository helpTransactionRepository;
 	private final HelpExperienceService experienceService;
 	private final DbActions dbActions;
-	private final String[][] closeSuggestionDetectors = {
-			{"close", "post"},
-			{"close", "thread"},
-			{"close", "question"},
-			{"problem","solv"},
-			{"issue","solv"},
-			{"thank"}
-	};
+	private final String[][] closeSuggestionDetectors = {{"close", "post"}, {"close", "thread"}, {"close", "question"}, {"problem", "solv"}, {"issue", "solv"}, {"thank"}};
```

Comment:
Why did you merge these lines?
If each detector has it's own line, it's more readable.

Reply:
To answer your question: my autoformat is set to doing things that way. Might update its preferences.